### PR TITLE
Adding all collections task with fixes from previous pull request.

### PR DESCRIPTION
**Pull request by martinmilev**
Adding all collections task with fixes from previous pull request.

### CODE REVIEW

File: src/com/clouway/collections/pagebean/Page.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.clouway.collections.pagebean;
+
+public class Page {
+  private String content;
```
Comment (mgenov) [inline src/com/clouway/collections/pagebean/Page.java]: Why you prefer content of the page to be represented as string  ?

File: src/com/clouway/collections/pagebean/PageBean.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    current++;
+    return show();
+  }
+
+  /**
+   * Returns the first page.
+   *
+   * @return
+   */
+  public String firstPage() {
```
Comment (mgenov) [inline src/com/clouway/collections/pagebean/PageBean.java]: Why you are returning string when method says that it returns firstPage ?

Comment (martinmilev) [inline src/com/clouway/collections/pagebean/PageBean.java]: There is no apparent reason that I can think of.
I wrote in it almost two weeks ago.
I could change all the navigation method to void.
And print only the show() method in the demo.

Comment (mgenov) [inline src/com/clouway/collections/pagebean/PageBean.java]: void will be even worse, cause printing will stay in production code ...

Comment (martinmilev) [inline src/com/clouway/collections/pagebean/PageBean.java]: Printing will be in the demo. 
Should I move the navigation in the PageBean and change all other methods to private.
```Java
public String navigation(String command) {
    if(command.equals(n)) {
      next();
      return show();
    }
    return null;
  }
```

Comment (mgenov) [inline src/com/clouway/collections/pagebean/PageBean.java]: If you move and navigation in PageBean this object would become more complex and will couple it with the navigation workflow.

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**